src/extensions/Statiq.Html/Globalization.cs

```diff
@@ -0,0 +1,531 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Dom.Html;
+using AngleSharp.Parser.Html;
+using Statiq.Common;
+
+namespace Statiq.Html
+{
+    public class Globalization : Module
```

Comment:
Can we rename this module to something that follows the [Verb][Noun] convention? Maybe `AddGlobalization`, `AddGlobalizationAttributes`, or `GlobalizeAttributes` (I'm open to other suggestions)?

Reply:
I renamed it to `AddRtlSupport`. That'll make it clear what it does. And many people (especially those whose first language is not English), might not now what is Globalization. This also follows the [Verb][Noun] convention. What do you think?